test/WebJobs.Extensions.DurableTask.Analyzers.Test/Binding/OrchestratorContextAnalyzerTests.cs

```diff
@@ -306,7 +383,85 @@ public static class HireEmployee
                 Severity = Severity,
                 Locations =
                  new[] {
-                            new DiagnosticResultLocation("Test0.cs", 11, 36)
+                            new DiagnosticResultLocation("Test0.cs", 13, 36)
+                     }
+            };
+
+            SyntaxNodeUtils.version = DurableVersion.V2;
+
+            VerifyCSharpDiagnostic(test, expectedDiagnostics);
+
+            VerifyCSharpFix(test, V2ExpectedFix);
+        }
+
+        // Tests SyntaxKind.GenericName
+        [TestMethod]
+        public void OrchestrationContext_V2_UsingList()
```

Comment:
I notice there is a lot of duplicate tests for v1 vs v2. Consider using parameterized tests for cases that apply to both.